package.json

```diff
@@ -58,6 +58,7 @@
     "scoped-property-store": "^0.16.2",
     "scrollbar-style": "^2.0.0",
     "season": "^5.1.2",
+    "selenium-webdriver": "^2.44.0",
```

Comment:
We don't want to ship this with Atom itself right?

If not, it should go in `build/package.json` then.